src/AliasCleaner.php

```diff
@@ -89,7 +89,7 @@ public function getCleanSeparators($string, $separator = NULL) {
 
     if (strlen($separator)) {
       // Trim any leading or trailing separators.
-      $output = trim($output, $separator);
+      $output = rtrim($output, $separator);
```

Comment:
$separator is not always a slash so I think we should keep the trim() here but ensure that the leading char is still a slash before returning $output.

Something like this might be enough:
```php
// If the separator is a slash, we need to re-add the leading slash dropped by the trim function.
if ($separator == '/') {
  $output = '/' . $output;
}
return $output;
```